DashAI/back/main.py

```diff
@@ -18,6 +19,23 @@
 app.mount(settings.API_V0_STR, api_v0)
 app.mount(settings.API_V1_STR, api_v1)
 
+# Load registries
+task_registry = get_task_registry()
+model_registry = get_model_registry()
+
+registry_dict_plugins = {"model": model_registry, "task": task_registry}
+
+
+# Temporary plugins system endpoint
+@app.get("/update_registry")
+def update_registry():
+    update_plugins(registry_dict_plugins)
+
+
+# Load installed plugins
+if settings.PLUGINS:
+    update_registry()
```

Comment:
You shouldnt be calling a FastAPI decorated path operation. Also if you need to startup the plugin system, it shouldnt be on FastAPI main. This file is intended to load endpoints and routes.